docs/user/intro/doctools.rst

```diff
@@ -65,6 +65,16 @@ Below is a list of popular documentation tools that you can use to write your do
         Written in
              :bdg-info:`rust`
 
+    .. grid-item-card::  VitePress
+        :link: vitepress.html
+
+        VitePress is a static site generator with a focus on performance and simplicity.
```

Comment:
Is it possible to add more spacing between each row? Similar to the space between columns

![Screenshot_2025-01-23_08-47-35](https://github.com/user-attachments/assets/7022bbbc-3a2f-4185-bb94-673128d4c6f4)

Reply:
Not sure, but that seems outside the scope for this PR?

Reply:
Yes.